### PR TITLE
[fix]: change the background color of the mostvisited websites

### PR DESCRIPTION
**Pull request by Keshraf**
### [fix]: change the background colour of the most visited websites

Due to non-uniform favicons of different websites, the design is taking. Hence, by matching the background of png favicons with the daily dev's background, the UI of the application would improve. I have added border colour as well to distinguish between adjacent icons.

### CODE REVIEW

**Comment by rebelchris** [comment]
@Keshraf Wow thanks a lot for putting in the work here 💪

We do have a in-house designer and quite a strict guideline for branding, these colors are not brand specified, but I've asked him to see if we should change this section based on your PR.

For future reference, when it comes to design related tasks its best to open a discussion or issue first.

Nevertheless thank you for your willingness and making the product better 💖

File: packages/extension/src/newtab/MostVisitedSites.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function MostVisitedSites(): ReactElement {
               href={topSite.url}
               rel="noopener noreferrer"
               className={classNames(
-                'focus-outline w-8 h-8 rounded-lg overflow-hidden bg-white',
+                'focus-outline w-8 h-8 rounded-lg overflow-hidden bg-[#0E1217] border-[#2D323B]',
```
Comment (rebelchris) [inline packages/extension/src/newtab/MostVisitedSites.tsx:27]: Asked our designed and he suggests to use the `secondary-divider` color.
You can use it like so:

```suggestion
                'focus-outline w-8 h-8 rounded-lg overflow-hidden border border-theme-divider-secondary',
```

**Review by rebelchris — CHANGES_REQUESTED** [review]
Hi @Keshraf ,

Thanks again for this great suggestion.
Our designer agrees that adding the border is a great addition.

However it should be our brand guideline colors (secondary)
See suggested fix.

**Review by rebelchris — APPROVED** [review]
Nice work, this looks great 👏

**Comment by rebelchris** [comment]
@Keshraf we recently had some changed in this file because of a new way of using it.
Would you mind resolving the issues 🙏

**Comment by rebelchris** [comment]
@Keshraf Sorry I meant to resolve the merge conflict issue if you would still like to help with this issue 🙏

**Comment by Keshraf** [comment]
@rebelchris Thank you for clarifying 😅. I would love to work on it.

**Pull request closed without merging**